src/time-map.scm

```diff
@@ -110,6 +110,22 @@
 	)
 )
 
+(define (gaze-at-face face-id-node)
```

Comment:
Please add documentation explaining what this routine does, why its needed, what it's arguments are, and how many times a second it might be called.